tests/unit/lms/services/rsa_key_service_test.py

```diff
@@ -12,6 +14,39 @@
 
 
 class TestAESService:
+    @freeze_time("2022-1-15")
+    def test_rotate(self, svc_generates, valid_keys, expired_keys, db_session):
+        """Keys created the 10th, max ages of 2 and 4 both expire valid and deleted expired on the 15th."""
+        target_keys = 5
+
+        # Assert what we think is the initial setup
+        assert not any((key.expired for key in valid_keys))
+        assert all((key.expired for key in expired_keys))
+        assert all((key in db_session for key in expired_keys))
+
+        svc_generates.rotate(target_keys, timedelta(days=2), timedelta(days=4))
+
+        assert all((key.expired for key in valid_keys))
+        assert not any((key in db_session for key in expired_keys))
+        assert db_session.query(RSAKey).filter_by(expired=False).count() == target_keys
+
+    @freeze_time("2022-1-11")
+    @pytest.mark.usefixtures("expired_keys")
+    def test_rotate_doesnt_expire_valid(self, svc_generates, valid_keys):
+        """Keys created the 10th, max age of 2,  doesn't expire the keys on the 11th."""
```

Comment:
```suggestion
        # Keys created the 10th, max age of 2, doesn't expire the keys on the
        # 11th.
```

I'm not clear on what this is saying